### PR TITLE
Transparent node markers

### PR DESCRIPTION
**Pull request by davide-f**
This pull request solves the issue discussed in #11 ; note that this pull request does not account for the pull request solving #7

### CODE REVIEW

File: src/SankeyPlots.jl [hunk 1]
```diff
@@ -167,18 +167,18 @@ In addition to [Plots.jl attributes](http://docs.juliaplots.org/latest/attribute
         @series begin
             primary := :false
             seriestype := :scatter
-            markersize := 0
+            markeralfa := 0
```
Comment (daschw) [inline src/SankeyPlots.jl]:
```suggestion
            markeralpha := 0
```

File: src/SankeyPlots.jl [hunk 2]
```diff
@@ -167,18 +167,18 @@ In addition to [Plots.jl attributes](http://docs.juliaplots.org/latest/attribute
...
-        
+
         # extend axes for labels
         if label_position in (:left, :right)
             x_extra = label_position === :left ? minimum(xlabs) - 0.4 : maximum(xlabs) + 0.5
             @series begin
                 primary := false
                 seriestype := :scatter
-                markersize := 0
+                markeralfa := 0
```
Comment (daschw) [inline src/SankeyPlots.jl]:
```suggestion
                markeralpha := 0
```

**Review by daschw — COMMENTED** [review]
Hi, thanks for your contributions! I suggest to use `markeralpha` instead of `markeralfa`.

File: src/SankeyPlots.jl (hunk 2 above)
Comment (davide-f) [reply src/SankeyPlots.jl]: ahahah sorry, sure! I copied the wrong text, sorry; I do have two copies and I copied the wrong one; in my source code the "alpha" one is used

**Pull request merged**